### ISSUE TITLE
Cannot use NMscanData with `use.input=FALSE`

### ISSUE THREAD

**Issue by billdenney** [philipdelff/NMdata#35]
I'm trying to make a reprex for this, but the quick issue is:

When I use the following to load several data files, I get the following error:

```r
NMdata::NMscanData("NONMEM/PK_akrv2/nash18.lst", use.input=FALSE)
# Warning in NMreadTab(meta[I, file], quiet = TRUE, tab.count = tab.count,  :
#   Duplicated column names found: DV. Cleaning.
# Warning in NMreadTab(meta[I, file], quiet = TRUE, tab.count = tab.count,  :
#   Duplicated column names found: ET_QCENTP. Cleaning.
# Error in file.exists(file.mod) : invalid 'file' argument
```

When I traced the error a bit, it appears that `file.mod` is `NULL`.  There are several places within `NMscanData()` where `file.exists(file.mod)` is called, and I don't immediately see which is causing the problem.

**Comment by philipdelff** [comment]
Thanks Bill!

This certainly looks like a bug. I will try to reproduce it with other examples so don't worry about the reprex for now. However, I am curious about whether file.mod points to an existing file or not. Does a file called nash18.mod exist next to nash18.lst (if not, try file.mod=identity)? Or did you modify file.mod either by the file.mod argument or using NMdataConf?

**Comment by billdenney** [comment]
Thanks for following up on this.

The file nash18.mod does exist, and I didn't try giving the file.mod option or using NMdataConf.  I'll give those a shot today.

**Comment by philipdelff** [comment]
Thanks, that narrows debugging. The input control stream is of no interest for this if you don't use input data so missing the file must be allowed. 

I am noticing you are getting warnings because of duplicated columns in output tables (which obviously has nothing to do with the error). That was changed to messages in NMdata 0.0.12 which was released to CRAN and MPN a couple of months ago and contains a few bugfixes (not that I know this one has been fixed). I will still try to reproduce the error, but you may want to try to update NMdata.

**Comment by philipdelff** [comment]
I can't reproduce this with NMdata 0.0.12. I think the bug must have been fixed between 0.0.11 and 0.012 where some of this was simplified. If you have the chance to update NMdata and re-run this, please let me know if it is resolved.

**Comment by billdenney** [comment]
This is fixed in 0.0.12.  Thanks!
